nilearn/datasets/tests/test_atlas.py

```diff
@@ -763,6 +763,52 @@ def test_fetch_atlas_pauli_2017(tmp_path, request_mocker):
         atlas.fetch_atlas_pauli_2017("junk for testing", data_dir)
 
 
+# TODO: remove this test after release 0.13.0
+def test_fetch_atlas_pauli_2017_deprecated_values(tmp_path, request_mocker):
```

Comment:
Do you think you could refactor this test and the previous one into a single one parametrized over atlas_type ?

If it makes sense

Reply:
I think that would complicate the tests. The assertions are different for each atlas_type. 

The previous test function tests if the function behaves as expected. 

This test tests if the warnings are correct  and also checks the functionality.

Reply:
ok fair enough: let's keep it like that then. 👍🏽